setup.py

```diff
@@ -37,11 +37,15 @@
     extras_require['all'].extend(reqs)
 
 setup_requires = [
+    'Babel>=1.3',
     'pytest-runner>=2.6.2',
 ]
 
 install_requires = [
+    'Flask-BabelEx>=0.9.2',
     'invenio-assets>=1.0.0',
+    'invenio-db>=1.0.2',
+    'invenio-access>=1.0.0',
```

Comment:
I'm about to say that you might actually not need `invenio-access` at all... The `AllowAll` permission is not depending on anything from `invenio_access`

Reply:
Exactly my thought, now that it works end to end, I 'll start killing dependencies.